app/assets/src/components/ui/containers/ContextPlaceholder.jsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
```

Comment:
The naming of this is a bit confusing. Why is it called a `ContextPlaceholder`? Sorry I think we talked about this but I can't remember. Can you add a comment at the top explaining why this is different from `Popup` (and why you needed to re-implement it)?

Reply:
Added a comment, but my thoughts on writing my own:
1. a popup is typically a very simple component with some text (that why the naming). Could use a better name than placeholder.
2. do now want a lot of the popup default styles (this is a invisible component for layout) and some positioning options were weird.
3. if we can implement simpler controls ourselves better (some semantic components are not that straightforward to use).
4. if a popup works on other use cases feel free to use it.

Reply:
I see...While a `<Popup>` usually just contains text, we do use it in components like `<ThresholdFilterDropdown>` for more complex popups so I think it is capable of handling that.

Let's keep ContextPlaceholder, but please do include in your comment an example of a use case where it's easier to use than Popup (for example, a simpler use case where you don't care about X) so the team can understand when it's more appropriate. Alternatively, if it's very specific to SamplesHeatmap, maybe move the component out of `/ui` and into that folder. Thanks!

Reply:
Edit: I see your point about using as an invisible component for layout only, as opposed to also providing styling. Didn't catch that earlier.